src/vs/platform/terminal/common/capabilities/capabilities.ts

```diff
@@ -5,6 +5,9 @@
 
 import { Event } from 'vs/base/common/event';
 import { IGenericMarkProperties, ISerializedCommandDetectionCapability } from 'vs/platform/terminal/common/terminalProcess';
+// Importing types is safe in any layer
+// eslint-disable-next-line code-import-patterns
+import type { IMarker } from 'xterm-headless';
```

Comment:
Not sure what this monaco editor checks error is about, just duplicate IMarker in this file if imports are causing problems